### ISSUE TITLE
[Feature Request]: Add a dedicated "New support doc" option

### ISSUE THREAD

**Issue by inaikem** [Automattic/bugomattic#185]
### "What": What is your idea?

We'd like to add a dedicated "Request a new WPcom support doc" option to the top-level "Report an issue" submenu and "Type" section radio buttons on Bugomattic. 

The link(s) would skip the user to show the Support Docs Content repo panel. It's currently possible to find this panel as follows:

1. Click 'Report an issue'
2. Select any of the three options under "Type"
3. Search for "doc" under Product and Feature.
4. See results for "WordPress.com > Support + Support Docs Content.

Screenshot:
![CleanShot 2024-05-08 at 11 02 44](https://github.com/Automattic/bugomattic/assets/66711744/0cf8e577-f486-4fde-ad64-0bbb92376796)

Note: This is the only "docs" related repo covered by Bugomattic, so as we see it, would not impact or set preference over any other division's documentation.

### "Why?": Why would this be impactful?

We have a specific focus on developer docs improvements and want to reduce friction for anyone who wants to submit docs suggestions, requests, or find the repo to make updates to content directly.

Cc: @donalirl in case he'd like to add more context.

### "How?" How should this feature work?

The proposed link(s) would skip the user to show the Support Docs Content repo panel. 

It's currently possible to find this panel as follows:

1. Click 'Report an issue'
2. Select any of the three options under "Type"
3. Search for "doc" under Product and Feature.
4. See results for "WordPress.com > Support + Support Docs Content.

Screenshot:
![CleanShot 2024-05-08 at 11 02 44](https://github.com/Automattic/bugomattic/assets/66711744/0cf8e577-f486-4fde-ad64-0bbb92376796)

Note: This is the only "docs" related repo covered by Bugomattic, so as I see it, making this addition would not impact or set preference over any other division's documentation.

Cc: @nerdysandy in case you want to add more context!

**Comment by donalirl** [comment]
> We have a specific focus on developer docs improvements and want to reduce friction for anyone who wants to submit docs suggestions, requests, or find the repo to make updates to content directly.
> 
> Cc: @donalirl in case he'd like to add more context.

Yes :) Just to mention that this specific GH repo is for the content on the English version of the /support site only. Developer docs have a different submission process: pfA0Z5-b-p2

**Comment by jeherve** [comment]
Do you mean to add a new option here?

<img width="684" alt="Screenshot 2024-05-09 at 10 55 39" src="https://github.com/Automattic/bugomattic/assets/426388/534d7854-9588-42a0-8b75-fc27a91558d8">

Since there are multiple divisions using Bugomattic, there would then have to be a new selection screen afterwards to allow folks to select the product they'd like to update.

As @donalirl mentioned above, even for WordPress.com there are different possible repos, so the flow would have to support multiple products.
